changelogs/fragments/23-hashi-vault-lookup-refresh.yaml

```
@@ -0,0 +1,13 @@
+bugfixes:
+  - hashi_vault - when a non-token authentication method like ldap or userpass failed, but a valid token was loaded anyway (via env or token file), the token was used to attempt authentication, hiding the failure of the requested auth method.
+  - hashi_vault - if used via `with_hashi_vault` and a list of (say) 4 secrets to retrieve, only the first one would be retrieved and returned 4 times.
+  - hashi_vault - error messages are not user friendly and don't contain the secret name ( https://github.com/ansible-collections/community.general/issues/54 )
```

```suggestion
  - hashi_vault - error messages are not user friendly and don't contain the secret name ( https://github.com/ansible-collections/community.general/issues/54 )
```
Did your change make the error messages not user friendly? If you improved them, you should say so :)

---

Should have been `now`, I don't know why I make this type so often, there are 2 keys between `w` and `t`! (fixed)